Aptoide/aptoide-client-v8
# [DT-1] Fix design

**What does this PR do?**

This PR aims at fixing the UI comments after review. Also took the chance to move the dt classes to its own submodule.

**Database changed?**

   No

**Where should the reviewer start?**

- [ ] EditorialView.kt

**How should this be manually tested?**

Open the app and check that the placeholders are changed and that the editorial tag is working on both themes (light and dark).

**What are the relevant tickets?**

  Tickets related to this pull-request: [DT-1](https://aptoide.atlassian.net/browse/DT-1)
  PR related to this pull-request: [DT-1] (https://github.com/Aptoide/aptoide-client-dt/pull/5)

**Questions:**

   Does this add new dependencies which need to be added? (Eg. new keys, etc.) 



**Code Review Checklist**


[DT-1]: https://aptoide.atlassian.net/browse/DT-1?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
